src/components/WebexMember/WebexMember.jsx

```diff
@@ -31,7 +32,10 @@ export default function WebexMember({
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="member-name">
```

Comment:
let's take out the subtitle from the `member-name` class, something like:
```html
<div.member>
    <avatar/>
    <div.description>
        <div.name/>
        <div.subtitle/>
    </div>
</div>
```
Feel free to use a better name than `description` if you find one.